Quotient/events/filesourceinfo.cpp

```diff
@@ -23,34 +23,36 @@ using namespace Quotient;
 QByteArray Quotient::decryptFile(const QByteArray& ciphertext,
                                  const EncryptedFileMetadata& metadata)
 {
-    if (QByteArray::fromBase64(metadata.hashes["sha256"_ls].toLatin1())
-        != QCryptographicHash::hash(ciphertext, QCryptographicHash::Sha256)) {
-        qCWarning(E2EE) << "Hash verification failed for file";
```

Comment:
do we intentionally not check the hash anymore?

Reply:
Uhm, good catch, no.